test/test_nn.py

```diff
@@ -5267,7 +5267,7 @@ def test_KLDivLoss_batch_mean(self):
 
         self.assertEqual(l, expected)
 
-    @unittest.skipIf(not (TEST_CUDNN and TEST_CUDNN_VERSION >= 7000), "needs cudnn >= 7.0")
+    @unittest.skipIf(not (TEST_CUDNN and (TEST_CUDNN_VERSION if TEST_CUDNN_VERSION else 0) >= 7000), "needs cudnn >= 7.0")
```

Comment:
This is kind of goofy can we just unconditionally set `TEST_CUDNN_VERSION` to zero if we don't have it set?